config/alpine/waagent.conf

```diff
@@ -75,8 +75,15 @@ OS.OpensslPath=None
 OS.SshDir=/etc/ssh
 
 # Enable or disable goal state processing auto-update, default is enabled
+# When turned off, it reverts to the pre-installed agent that comes with image
+# Added new option AutoUpdate.UpdateToLatestVersion, and encourage users to transition to this new option
```

Comment:
Once we know release #, i think it may be helpful to add which release added support for AutoUpdate.UpdateToLatestVersion. 

Also maybe we can mention why we encourage users to transition to this new option (for example, if you revert to pre-installed agent you lose many bug fixes, etc.)